tests/org.eclipse.n4js.ide.tests/src/org/eclipse/n4js/ide/tests/server/package-info.java

```diff
@@ -0,0 +1,227 @@
+	 */
+	static int DEFAULT_PROJECT;
+
+	/**
+	 * If defined, a default yarn workspace is used for all tests of one
+	 * {@link org.eclipse.n4js.ide.tests.server.AbstractIdeTest}. However, an additional module is added to this default
+	 * workspace. This additional module contains the actual test case that will refer to projects and modules of its
+	 * default workspace.
+	 * <p>
+	 * Defining a default yarn workspace can be done using the convenient xtend list, pair and string literal notation.
+	 * This notation will also be used in the following examples.
+	 *
+	 * <b>Simple default yarn workspace</b>
+	 *
+	 * <pre>
+	 * override final List<Pair<String, String>> getDefaultTestModules() {
+	 * 		return #[
+	 * 			"P1*" -> #[
+	 * 				"M1" -> '''
+	 * 					import "XY" from "LibXY";
+	 * 					const xy = new XY();
+	 * 				'''],
+	 * 			"P2" -> #[
+	 * 				"LibXY" -> '''
+	 * 					export public class XY {}
+	 * 				''']
+	 * 		];
+	 * }
+	 * </pre>
+	 */
+	static int DEFAULT_WORKSPACE;
+
+	/**
+	 * Module names used in default test projects or workspace can be given either with extension or without. The
+	 * default extension is {@code n4js} if no extension (which must be one of
+	 * {@link org.eclipse.n4js.N4JSGlobals#ALL_N4_FILE_EXTENSIONS}) was given.
+	 * <p>
+	 * In case the module name is {@code package.json}, see {@link #PACKAGE_JSON}.
+	 * <p>
+	 * In case the string starts with the reserved string
+	 * {@link org.eclipse.n4js.ide.tests.server.TestWorkspaceCreator#NODE_MODULES}, see {@link #PROJECT_NODE_MODULES} or
+	 * {@link #WORKSPACE_NODE_MODULES}.
+	 */
+	static int MODULE_NAMES;
+
+	/**
+	 * {@code package.json} files are created automatically. In case this should be overridden by a custom file, use
+	 * {@code package.json} as a module name and define its contents.
+	 *
+	 * <p>
+	 * <b>Simple example</b>
+	 *
+	 * <pre>
+	 * override final List<Pair<String, String>> getDefaultTestModules() {
+	 * 		return #[
+	 * 			NODE_MODULES + "@n4jsd/SomeNPM" + SRC + "index.n4jsd" -> '''
+	 * 				export public external class A1 {}
+	 * 				''',
+	 * 			NODE_MODULES + "@n4jsd/SomeNPM" + SRC + PACKAGE_JSON  -> '''
+	 * 				{
+	 * 					"name": "@n4jsd/SomeNPM",
+	 * 					"version": "0.0.1",
+	 * 					"n4js": {
+	 * 						"projectType": "definition",
+	 * 						"definesPackage": "someNPM",
+	 * 						"sources": {
+	 * 							"source": [
+	 * 								"src"
+	 * 							]
+	 * 						}
+	 * 					}
+	 * 				}'''
+	 * 		];
+	 * }
+	 * </pre>
+	 *
+	 * The example above shows how a default project that only defines one project inside the node_modules folder. Since
+	 * this project is of project type {@code definition}, its package.json file is defined explicitly and overrides the
+	 * one that would have been generated. Note that the reserved string
+	 * {@link org.eclipse.n4js.ide.tests.server.TestWorkspaceCreator#SRC} is misleading here: the package.json file will
+	 * be located in the base directory of the project.
+	 */
+	static int PACKAGE_JSON;
```

Comment:
Add a link to constant `TestWorkspaceCreator#PACKAGE_JSON` at beginning of this element's text. (same as for the other properties below)